### PR TITLE
Add view elements to WhatIsNewView

### PR DESCRIPTION
**Pull request by Gio2018**
Ref #14581

Adds base view elements to the "What's New / Feature Announcements" page.

**Notes:**
**1. the table view that will contain the actual announcements, as well as the "Find out more" button is not populated yet. At the moment, you will just see a grey rectangle in place of it; the grey background will be removed in future PRs.
2. The modal size on iPad has not been customized at the moment and it's using the default (which on iOS 12 would be full screen).**

To test:
1. Go to Me -> App Settings -> What's new in WordPress
2. Verify that the title "What's New in WordPress" appears and the version number is consistent to the actual version. For reference, see design in #14581 and screenshots below. 
3. The Continue button should dismiss the modal.

iPhone | iPad
--- | ---
whatsnewiphone | whatsnewipad

PR submission checklist:

- [x] ~~I have considered adding unit tests where possible.~~ Not yet added.
- [x] ~~I have considered adding accessibility improvements for my changes.~~ Not yet added.
- [x] ~~I have considered if this change warrants user-facing release notes and have added them to `RELEASE-NOTES.txt` if necessary.~~ Not released.

### CODE REVIEW

**Review by osullivanchris — APPROVED** [review]
Looks great so far!

File: WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+
+    func makeWhatIsNewViewController() -> WhatIsNewViewController {
+        return WhatIsNewViewController(whatIsNewViewFactory: makeWhatIsNewView)
+    }
+
+    private func makeWhatIsNewView() -> WhatIsNewView {
+
+        var version = ""
+        if let versionNumber = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
+            version = NSLocalizedString("Version ", comment: "Description for the version label in the What's new page.") + versionNumber
```
Comment (ScoutHarris) [inline WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift:12]: What do you think about making a `private enum` for all the `NSLocalizedString`s in this method? It'd make it more readable. For example, I almost missed the tacked on `+ versionNumber` here as it was kind of obfuscated by the string definition.

File: WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+extension WPTabBarController {
+
+    func makeWhatIsNewViewController() -> WhatIsNewViewController {
+        return WhatIsNewViewController(whatIsNewViewFactory: makeWhatIsNewView)
+    }
+
+    private func makeWhatIsNewView() -> WhatIsNewView {
+
+        var version = ""
+        if let versionNumber = Bundle.main.infoDictionary?["CFBundleShortVersionString"] as? String {
```
Comment (ScoutHarris) [inline WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift:11]: We have a `NSBundle+VersionNumberHelper` that does this. Maybe you can use `Bundle.main.shortVersionString()`?

File: WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift (hunk 1 above)
Comment (Gio2018) [reply WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift]: Totally more readable! Thanks for pointing this out!

File: WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift (hunk 2 above)
Comment (Gio2018) [reply WordPress/Classes/ViewRelated/What's New/WPTabBarController+WhatIsNew.swift]: yes absolutely, I had missed this helper, thank you!

**Pull request merged**